dali/python/nvidia/dali/plugin/jax.py

```diff
@@ -23,8 +25,8 @@
     "DALI JAX support requires Python 3.8 or above"
 
 
-assert LooseVersion(jax.__version__) >= LooseVersion('0.4.10'), \
-    "DALI JAX support requires JAX 0.4.10 or above"
+assert LooseVersion(jax.__version__) >= LooseVersion('0.4.11'), \
+    "DALI JAX support requires JAX 0.4.11 or above"
```

Comment:
Some of the APIs that we need to use changed names from 0.4.10 to 0.4.11. I do not think it makes sense to maintain two code paths at this point since there is no one using it yet. That's why I just bumped the required version.